src/NuGet.Core/Microsoft.Build.NuGetSdkResolver/Microsoft.Build.NuGetSdkResolver.csproj

```diff
@@ -4,7 +4,7 @@
 
 
   <PropertyGroup>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);netstandard2.1</TargetFrameworks>
```

Comment:
I'd recommending conditioning the value of TargetFrameworksLibrary on some property instead of manually writing netstandard2.1. 

Chat with me if you unsure what I'm suggesting.

Reply:
Thanks! Changed accordingly.